packages/extension-ui/src/partials/MenuAdd.tsx

```diff
@@ -11,23 +11,31 @@ import plusIcon from '../assets/plus.svg';
 import qrIcon from '../assets/qr.svg';
 import seedIcon from '../assets/secret.svg';
 import { AccountContext, Link, MediaContext, Menu, MenuDivider, MenuItem, Svg } from '../components';
+import useIsPopup from '../hooks/useIsPopup';
 import useTranslation from '../hooks/useTranslation';
-import { jsonRestoreWindowOpen } from '../messaging';
+import { windowOpen } from '../messaging';
 
 interface Props extends ThemeProps {
   className?: string;
   reference: React.MutableRefObject<null>;
 }
 
-const isPopup = window.innerWidth <= 480;
```

Comment:
squashed what looks like a a bug along the way, was set as 480 here and 560 below. Built a hook for this now.

Reply:
2nd time... fixed something similar in the last round :(